tests/unit/result_handlers/test_text.py

```diff
@@ -16,11 +16,13 @@
 
 from pandas import DataFrame
 
+from data_validation.consts import VALIDATION_STATUSES
+
 SAMPLE_CONFIG = {}
 SAMPLE_RESULT_DATA = [
-    [0, 1, 2, 3, "Column", "source", "target"],
-    [4, 5, 6, 7, "Column", "source", "target"],
-    [8, 9, 10, 11, "Column", "source", "target"],
+    [0, 1, 2, 3, "Column", "source", "target", "success"],
```

Comment:
After changing the TextResultHandler() to have a default value of None, you won't need to change this test data or the `test_columns_to_print` test.

Reply:
I needed to change this data to have at least one failed result for the new unit test